build.gradle

```diff
@@ -5,8 +5,10 @@ plugins {
 allprojects {
     repositories {
         jcenter()
+        mavenCentral()
```

Comment:
Why do we need to add the mavenCentral repo. As per my knowledge jcenter is a superset of maven central.

Reply:
I was facing some issues but then I got rid of them .. I should have removed it but forgot..
good catch, will remove it